test/library/memory-store-test.ts

```diff
@@ -132,4 +132,16 @@ describe('memory store test', () => {
 			clearTimeout(realTimeoutId)
 		}
 	})
+
+	it('When calling init it should reset interval when already initialized', async () => {
```

Comment:
nit: could you please rename the test to: "should reset the interval incase it already exists"